umm-spec-lib/src/cmr/umm_spec/xml_to_umm_mappings/iso19115_2.clj

```diff
@@ -234,6 +238,7 @@
       :TilingIdentificationSystems (tiling/parse-tiling-system md-data-id-el)
       :TemporalExtents (or (seq (parse-temporal-extents doc extent-info md-data-id-el))
                            (when sanitize? su/not-provided-temporal-extents))
+      :CollectionDataType (value-of (select doc collection-data-type-xpath) ".")
```

Comment:
I assume collection data type is an enumeration of science and near-real-time or something like that? So even though we're not validating here, we'll throw an error during UMM validation if the value isn't valid?

Sorry it's been awhile since I've reviewed a UMM change. 

I also assume the reason for no tests is that the generator tests and the change to iso19115_expected_conversion.clj are sufficient.